lib/tasks/audit.rake

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+desc 'Audit accessioned WARCs against WARCs available from WASAPI provider'
+task :audit, %i[collection_druid] => :environment do |_task, args|
```

Comment:
Since WRA knows what collections are inside does it makes sense to have the rake task simply audit all its collections?

Reply:
Or have that as an option?